src/styles/components/charts/styles.less

```diff
@@ -23,7 +23,7 @@
     .grid-graph .tick,
     .axis line {
       fill: none;
-      shape-rendering: crispedges;
+      shape-rendering: crispEdges;
```

Comment:
I don't know why this is related, but this was breaking the build.

Reply:
update to `stylelint` most likely. we need to go to > 7.8.0 to get support for `fr` units to use CSS grid.